### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

**Comment by stof** [comment]
CI is still red

**Comment by nicolas-grekas** [comment]
I know, just wait a bit :)

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ language: php
 
 sudo: false
 
+cache:
+    directories:
+        - vendor
```
Comment (stof) [inline .travis.yml]: this can lead to weird things in composer. If the goal is to cache the PHPUnit download, put it outside `vendor` IMO

Comment (nicolas-grekas) [inline .travis.yml]: the component is small enough to not trigger strange things IMHO. We'll see :)

**Pull request merged**